src/Frost/PandocRun.hs

```diff
@@ -0,0 +1,38 @@
+module Frost.PandocRun where
+
+import Prelude hiding (readFile, writeFile)
+import Text.Pandoc hiding (trace)
+import Polysemy
+import Polysemy.Input
+import Polysemy.Output
+import Polysemy.Error
+import PolysemyContrib
+
+runInputPandoc :: (
```

Comment:
Moving of these functions should be done in a separate commit, because it's now harder to see what changed aside from the names

Reply:
It was move and rename. Nothing else changed.

Reply:
Should it be first move (commit 1) then rename (commit 2)?